src/database/models/team_item.rs

```diff
@@ -4,7 +4,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize)]
 pub struct Team {
     /// The id of the team
-    pub id: i32,
+    pub id: i64,
     /// A list of the members of the team
     pub members: Vec<TeamMember>,
```

Comment:
This should be a list of the ids of team members, since it is the database representation.  Although, since this is a representation of the database models, should it even be here?  It would require a second SQL query to retrieve, and doesn't match the structure of the teams table.  We should remove all of the `Vec`s in the database models, since those are left over from the mongodb model.